pandas/core/nanops.py

```diff
@@ -633,10 +633,31 @@ def _kendall(a, b):
     def _spearman(a, b):
         return spearmanr(a, b)[0]
 
+    def _biweight_midcorrelation(a, b):
+        a_median = a.median()
+        b_median = b.median()
+
+        # Median absolute deviation
+        a_mad = (a - a_median).abs().median()
```

Comment:
note: `abs` is actually a (builtin) function, too, if that makes your code look any cleaner